_data/talks.yml

```diff
@@ -3,374 +3,437 @@
 - title: PyCon India 2020 Opening Address
   track: all
   date: 2nd Oct
-  time: 12:00 noon
+  time: 12:00 IST
```

Comment:
@anistark -- Can we remove IST's from the time everywhere? We would have this written at the top somewhere. Maybe I wasn't very clear on my earlier comment. :slightly_smiling_face: